app/models/topic_embed.rb

```diff
@@ -27,7 +27,7 @@ def self.imported_from_html(url)
   end
 
   # Import an article from a source (RSS/Atom/Other)
-  def self.import(user, url, title, contents, category_id: nil, cook_method: nil)
+  def self.import(user, url, title, contents, category_id: nil, cook_method: nil, tags: nil)
```

Comment:
If we're planning to use this from a plugin, can we expose this method via the plugin API instead? That way, we'll at least be able to better manager deprecation on this method in the future.

Reply:
IMO is generally useful being able to add a category and tags to an imported topic in the codebase.

Reply:
It may be useful but there isn’t anywhere in core that is using this right now.

Reply:
Can you expand a bit on this?

Do you want me to create a copy of this method under `/lib/plugin/instance.rb` but with the extra flag parameter and call that new version from 

https://github.com/discourse/discourse-rss-polling/blob/3339e5983fd13f95167ef143283f0026b892171c/app/jobs/jobs/discourse_rss_polling/poll_feed.rb#L38

?

Reply:
I don't think we should hold up this PR any longer. Please merge. @tgxworld can you follow up on dev with a post about what you are looking for?